File: cf/div4/944/c/main.cpp
#include<bits/stdc++.h>
using namespace std;

void solve(){
  int a, b, c, d;
  cin >> a >> b >> c >> d;
  if(a>b) swap(a, b);
  if(c>d) swap(c, d);
  if((a<c&&c<b&&b<d&&d<a+12)||(a<c+12&&c+12<b&&b<d&&d<a+12)) cout << "Yes" << endl;
  else cout << "No" << endl;
  return;
}

int main(){
  int T;
  cin >> T;

  while(T--){
    solve();
  }
}

File: cf/div4/944/c/main.py
def solve():
  a, b, c, d = map(int, input().split())
  if a>b:
    a,b = b, a
  if c>d:
    c,d = d, c
  #print(a, b, c, d)
  if (a<c<b and c<b<d) or (a<d<b and c<a<d):
    print("Yes")
  else:
    print("No")



T = int(input())
for _ in range(T):
  solve()